pkg/api/scylla/v1/types_cluster.go

```diff
@@ -408,6 +433,11 @@ type BackupTaskSpec struct {
 	// +optional
 	Retention int64 `json:"retention,omitempty" mapstructure:"retention,omitempty"`
 
+	// retention is the number of days for which a backup is stored. The next scheduled backup initiates a purge of expired backups.
+	// Can be used in tandem with `retention`.
+	// +optional
+	RetentionDays *int `json:"retentionDays,omitempty" mapstructure:"retention_days,omitempty"`
```

Comment:
But then there is a difference between operator and SM.
Running `sctool backup --retention-days 3` and corresponding (in a direct way) operator definition results in a different retention policy. Of course it's up to operator team to decide that, but I just wanted to highlight that.

Reply:
> imo the only reasonable thing to do would be to drop the defaults on our side altogether

you can stop defaulting the field but you can't make it required

We should not use defaults, if we want the manager to accept optional values. But that ship has half sailed. We default at admission and after that point the values are stored and always used as explicitly specified.

And defaulting or the lack of shall not render object invalid.

Reply:
I'll verify which defaults could be dropped safely, but that's for a separate PR anyway, since this ended up just adding cron and timezone. So let's get back to it once all the fixes in queue land, I don't want to introduce extra variables.